src/components/Filters/Filters.tsx

```diff
@@ -1,22 +1,43 @@
-import React, { useState, useRef } from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.css';
 import Button from '../Button';
-import { CheckboxField, FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+import { FiltersCheckboxField } from '../FiltersCheckboxField/FiltersCheckboxField';
+import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
 import Icon from '../Icon';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Text to be placed in the button that activates the Filters Drawer
    */
-  checkboxFields: CheckboxField[];
+  triggerText?: string;
+  /**
+   * Button status variant that indicates if filters had been selected.
+   */
+  hasFilters?: boolean;
+  /**
+   * Input components, input component fields, or relevant information that will be displayed in the filters drawer.
```

Comment:
Sorry to be all nitty with these comments but I think I would assume "input" meant only text input fields as opposed to checkboxes and whatnot. How about "form controls"? Like `Form controls, form fields, or other relevant information that will be displayed in the filters drawer.`.

Reply:
No need to be sorry :D 
Definitely a better comment, changed